portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/DatasetsFilters.testcase

```diff
@@ -0,0 +1,214 @@
+
+		FrontendDataSetAdmin.deleteAllDatasetEntries();
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-183476. Confirm that all fields in the schema are displayed"
+	@priority = 5
+	test AssertAllFieldsAreDisplayedInSchema {
+		task ("And clicks to add a new filter") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("asserts all fields in the schema are displayed") {
+			Click(locator1 = "FrontendDataSetAdmin#FILTER_BY_SELECT");
+
+			FrontendDataSetAdmin.assertSearchFilters(key_filterList = "creator,externalReferenceCode,id,status,createDate,fieldName,from,modifiedDate,name,r_fdsViewFDSDateFilterRelationship_c_fdsViewId,rendererType,r_fdsViewFDSDateFilterRelationship_c_fdsViewERC,to,type");
+		}
+	}
+
+	@description = "LPS-183476. Confirm that when 'Name Field' is not provided, it is filled with the same name as field name."
+	@priority = 3
+	test AssertNameIsFilledByDefault {
+		task ("And adds a new filter") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("And Choose an ID in the Filter By field") {
+			FrontendDataSetAdmin.selectFilters(filterType = "id");
+		}
+
+		task ("And Clicks Save") {
+			Button.clickSave();
+		}
+
+		task ("Then Confirm that the name of the saved filter is the same name as the chosen field") {
+			AssertElementPresent(
+				chosenFilter = "id",
+				key_tableColumn = 1,
+				key_tableRow = 1,
+				locator1 = "FrontendDataSetAdmin#FILTER_TABLE_CELL");
+		}
+	}
+
+	@description = "LPS-183476. Confirm that 'No default filters were created.' and 'Start creating a filter to display specific data' messages are displayed"
+	@priority = 3
+	test CanAssertNoDefaultFilters {
+		task ("Then asserts the message: 'No default filters were created. Start creating a filter to display specific data' is displayed") {
+			AssertTextEquals.assertText(
+				locator1 = "Message#EMPTY_STATE_INFO",
+				value1 = "No default filters were created.");
+
+			AssertTextEquals.assertText(
+				locator1 = "Message#EMPTY_STATE_TEXT",
+				value1 = "Start creating a Filter to display specific Data.");
+		}
+	}
+
+	@description = "LPS-183476. Confirm that the New filter modal is closed and the changes are not applied when canceling on Cancel button"
+	@priority = 3
+	test CanCancelFilterCreationOnCancelButton {
+		task ("And clicks to add a new filter") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("Then click on the "Cancel button"") {
+			Button.clickCancel();
+		}
+
+		task ("And assert the New filter modal is closed and the changes are not applied") {
+			AssertElementNotPresent(
+				key_modalTitle = "New Filter",
+				locator1 = "Button#CLOSE_MODAL");
+
+			AssertTextEquals.assertText(
+				locator1 = "Message#EMPTY_STATE_INFO",
+				value1 = "No default filters were created.");
+		}
+	}
+
+	@description = "LPS-183476. Confirm that the New filter modal is closed and the changes are not applied when canceling on Close (X) button"
```

Comment:
Put the correct LPS--> LPS-176184 (in Poshi, we should put the LPS of the original story/bug)